src/index.html

```diff
@@ -2,11 +2,296 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#"
+         class="header__logo"
+      >
+        <img src="images/logo.png" alt="Moyo-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a href="#" class="nav__item is-active">Apple</a>
+          </li>
+          <li class="nav__link">
+            <a href="#" class="nav__item">Samsung</a>
+          </li>
+          <li class="nav__link">
+            <a href="#" class="nav__item">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__link">
+            <a
+              href="#"
+              class="nav__item"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__link">
+            <a href="#" class="nav__item">гаджеты</a>
+          </li>
+          <li class="nav__link"><a href="#" class="nav__item">планшеты</a>
+          </li>
+          <li class="nav__link">
+            <a href="#" class="nav__item">фото</a>
+          </li>
+          <li class="nav__link">
+            <a href="#" class="nav__item">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog-wrapper">
+        <div class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac-image"
+            class="card__img"
+          >
+          <p class="card__card-content">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          <p class="card__card-productCode">Код товара: 195434</p>
+          <div class="card__reviews">
+            <div class="card__reviews-stars stars-count-4">
+              <div class="star"></div>
```

Comment:
```suggestion
            <div class="stars stars--count-4">
              <div class="stars__star"></div>
```
Let's make a separate `stars` block, because it has it's own logic and use it's 'stars' inside as an elements